### PR TITLE
Kucoin :: fix parseOrder

### PR DESCRIPTION
**Pull request by carlosmiei**
- fix https://github.com/ccxt/ccxt/issues/16669

Demo:

```
 p kucoin fetchOrder "63d690536b00ed00012aa0df" 
Python v3.10.8
CCXT v2.7.8
kucoin.fetchOrder(63d690536b00ed00012aa0df)
{'amount': 0.1,
 'average': 95.135,
 'clientOrderId': '54827f65-acc6-4dc2-bf70-856696646b3c',
 'cost': 9.5135,
 'datetime': '2023-01-29T15:27:15.636Z',
 'fee': {'cost': 0.0095135, 'currency': 'USDT'},
 'fees': [{'cost': 0.0095135, 'currency': 'USDT'}],
 'filled': 0.1,
 'id': '63d690536b00ed00012aa0df',
 'info': {'cancelAfter': 0,
          'cancelExist': False,
          'channel': 'API',
          'clientOid': '54827f65-acc6-4dc2-bf70-856696646b3c',
          'createdAt': 1675006035636,
          'dealFunds': '9.5135',
          'dealSize': '0.1',
          'fee': '0.0095135',
          'feeCurrency': 'USDT',
          'funds': '0',
          'hidden': False,
          'iceberg': False,
          'id': '63d690536b00ed00012aa0df',
          'isActive': False,
          'opType': 'DEAL',
          'postOnly': False,
          'price': '0',
          'remark': None,
          'side': 'buy',
          'size': '0.1',
          'stop': '',
          'stopPrice': '0',
          'stopTriggered': False,
          'stp': '',
          'symbol': 'LTC-USDT',
          'tags': 'partner:ccxt',
          'timeInForce': 'GTC',
          'tradeType': 'TRADE',
          'type': 'market',
          'visibleSize': '0'},
 'lastTradeTimestamp': None,
 'postOnly': False,
 'price': 95.135,
 'reduceOnly': None,
 'remaining': 0.0,
 'side': 'buy',
 'status': 'closed',
 'stopPrice': 0.0,
 'symbol': 'LTC/USDT',
 'timeInForce': 'GTC',
 'timestamp': 1675006035636,
 'trades': [],
 'triggerPrice': 0.0,
 'type': 'market'}
```
```
n kucoin createOrder LTC/USDT limit buy 0.1 50  
Debugger attached.
2023-01-29T15:26:33.956Z
Node.js: v17.9.0
CCXT v2.7.8
kucoin.createOrder (LTC/USDT, limit, buy, 0.1, 50)
2023-01-29T15:26:36.278Z iteration 0 passed in 376 ms

{
  info: { orderId: '63d6902c6b00ed00012a5f60' },
  id: '63d6902c6b00ed00012a5f60',
  clientOrderId: undefined,
  symbol: 'LTC/USDT',
  type: undefined,
  timeInForce: undefined,
  postOnly: undefined,
  side: undefined,
  amount: undefined,
  price: undefined,
  stopPrice: undefined,
  triggerPrice: undefined,
  cost: undefined,
  filled: undefined,
  remaining: undefined,
  timestamp: undefined,
  datetime: undefined,
  fee: { currency: undefined, cost: undefined },
  status: undefined,
  lastTradeTimestamp: undefined,
  average: undefined,
  trades: [],
  fees: [ { currency: undefined, cost: undefined } ],
  reduceOnly: undefined
}
2023-01-29T15:26:36.278Z iteration 1 passed in 376 ms
```

### CODE REVIEW

File: js/kucoin.js [hunk 1]
```diff
@@ -1895,10 +1895,13 @@ module.exports = class kucoin extends Exchange {
         if (cancelExist) {
             status = 'canceled';
         }
+        if (status === undefined) {
+            status = 'closed';
+        }
```
Comment (xmatthias) [inline js/kucoin.js:1900]: @carlosmiei what's the reason to parse "nothing" as "closed"? 

this makes every `createOrder` call automatically return the status "closed" - which is wrong  unless it's a market order (and even then, it's information that ccxt invented, not  something the exchange provided).

Comment (nuaimat) [reply js/kucoin.js:1900]: @carlosmiei kindly reminder for the above question.

Comment (samgermain) [reply js/kucoin.js:1900]: > @carlosmiei what's the reason to parse "nothing" as "closed"?
> 
> this makes every `createOrder` call automatically return the status "closed" - which is wrong unless it's a market order (and even then, it's information that ccxt invented, not something the exchange provided).

@carlosmiei This is true, I going to remove this addition in a PR

Comment (carlosmiei) [reply js/kucoin.js:1900]: @xmatthias @samgermain sorry guys I missed this comment

**Pull request merged**